apps/user-profile/src/app/pipes/validate-expiration.pipe.ts

```diff
@@ -7,6 +7,6 @@ import { Attribute } from '@perun-web-apps/perun/openapi';
 })
 export class ValidateExpirationPipe implements PipeTransform {
   transform(expirationAttribute: Attribute): string {
-    return (expirationAttribute?.value as unknown as string) ?? 'never';
+    return String(expirationAttribute?.value) ?? 'never';
```

Comment:
```suggestion
    return expirationAttribute?.value ? String(expirationAttribute?.value) : 'never';
```
![image](https://user-images.githubusercontent.com/32871167/171027972-d2e1e330-b1fa-487f-ac84-9c2ea4fbc847.png)
We can never use String() constructor in if statement, because it will just translate the undefined od translate into string

Reply:
Dunno if my suggestion will be liked by the eslint, but it cant stay this way

Reply:
Wait, does this mean that we cannot use the String() constructor anywhere?